test/shared/urls.test.ts

```diff
@@ -36,6 +36,19 @@ describe("shared/commands/parsers", () => {
       expect(parsers.searchUrl('std::vector', config))
         .to.equal('https://google.com/search?q=std%3A%3Avector');
     });
+
+    it('localhost urls', () => {
+      expect(parsers.searchUrl('localhost', config))
+	.to.equal('http://localhost');
+      expect(parsers.searchUrl('http://localhost', config))
+	.to.equal('http://localhost');
```

Comment:
`http://localhost` -> `http://localhost/`

Reply:
thank you for the hacking trick，i didn't manage to run test at first.